engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3366,6 +3370,28 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
 
   }
 
+  private class GetContainerStackTrace implements Command
```

Comment:
Class name should correspond to the command name. e.g. GetContainerStacks. Or change the command to get-container-stacktrace